src/storage/statistics/base_statistics.cpp

```diff
@@ -172,6 +172,7 @@ unique_ptr<BaseStatistics> BaseStatistics::Deserialize(Deserializer &source, Log
 		result = StructStatistics::Deserialize(reader, move(type));
 		break;
 	case PhysicalType::LIST:
+	case PhysicalType::MAP:
```

Comment:
Is the `PhysicalType::MAP` necessary/used? Is it `MAP` not just a `LIST` now instead of a `STRUCT`?

Reply:
Ha! Yes, good idea! PhysicalType::MAP is actually not necessary at all, so I took it out